blocks/search/search.js

```diff
@@ -67,12 +69,9 @@ function clearSearchResults(block) {
 }
 
 /**
- * Clear search entirely; remove from URL params and results markup.
- * @param {HTMLElement} block The main search block element.
+ * Clear search from URL params and update history.
  */
-function clearSearch(block) {
-  clearSearchResults(block);
```

Comment:
This function name was confusing; renamed it and refactored a little so these two separate functions are separate.